chevah/keycert/tests/test_ssh.py

```diff
@@ -1208,6 +1207,13 @@ def test_fromString_PUBLIC_SSHCOM_RSA_invalid_payload(self):
 
         self.assertKeyParseError(content)
 
+        # This will parse without errors but the blob is invalid
```

Comment:
this needs a separate test, with a dedicated docstring describing the targeted test scenario